backend/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+	"xkcd/comics/service"
+)
+
+func main() {
+    fmt.Println("Hello, World! Welcome to XKCD go bootcamp")
+
+    latestComicNum, err := service.GetLatestComicNumber()
+    if err != nil {
+        fmt.Printf("Error retrieving latest comic number: %v\n", err)
+        return
+    }
+
+    // Retrieve comics sequentially
+    seqComics, seqDuration := service.RetrieveComicsSequentially(1, latestComicNum)
+    fmt.Printf("Number of comics retrieved is: %d", len(seqComics))
+    fmt.Printf("Time taken to retrieve all comics sequentially: %v\n", seqDuration)
+
+    // Retrieve comics concurrently
+    conComics, conDuration := service.RetrieveComicsConcurrently(1, latestComicNum)
+    fmt.Printf("Number of comics retrieved is: %d", len(conComics))
+    fmt.Printf("Time taken to retrieve all comics concurrently: %v\n", conDuration)
+
+    // Retrieve information about a random comic
+    rand.NewSource(time.Now().UnixNano())
```

Comment:
what go version are you using? I think this approach is now deprecated and you no longer need to do the rand.NewSource thing.